roles/server/templates/freefeed-job.j2

```diff
@@ -9,7 +9,7 @@ output=$(docker run -t \
                 -e NODE_ENV=production \
                 -e NODE_CONFIG_ENV={{ freefeed_env }} \
                 --network freefeed \
-                freefeed/freefeed-server:"${version}" \
+                ghcr.io/freefeed/freefeed-server:"${version}" \
```

Comment:
can't see packages fresher than 2019 in ghcr.io

Reply:
because we've been using hub.docker.com, new images should appear after https://github.com/FreeFeed/freefeed-server/pull/614 and https://github.com/FreeFeed/freefeed-react-client/pull/1591